README.md

```diff
@@ -151,6 +151,26 @@ make e2e-local-run
 ```sh
 make push IMAGE=quay.io/myorg/3scale-operator VERSION=test
 ```
+## Deploy to OpenShift 4 using OLM
```

Comment:
If I understand correctly, this is to deploy it to OpenShift 4 on OLM ONLY for testing. I'd clarify that in the title, because at the beginning it semed that it was to deploy it in OpenShift 4 if you want to use it.
We should be the only ones able to peform an official "deploy" of the operator in OLM.
Maybe call this "Upload" or "Provision" instead of "Deploy"?

Reply:
First line of the section says:
`To install this operator on OpenShift 4 for end-to-end testing....`
It is pretty clear. 

I think we can re-structure documentation to have one section about `deploying operator manually` and `Deploy to OpenShift 4 using OLM`. We can state explicitly that is it for testing and development, but IMO it is not necessary. Deploying this operator on public catalogs (RH's OpenShift OperatorHub or community  Openshift catalog) is totally out of scope. Anyway, this is out of scope of this PR.